### PR TITLE
NCSD-1947: Adds script to create initial project.

### PR DESCRIPTION
**Pull request by thelious1974**
Adds readme with outline instructions

### CODE REVIEW

File: Scripts/New-InitialDotNetCoreProjects.ps1 [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+function New-BasicProject {
+    param(
+        [Parameter(Mandatory=$true)]
+        [string] $Name    
+    )
+
+    $testProject = $Name + ".UnitTests"
+
+    & "dotnet" "new" "web" "--name" $Name
```
Comment (NickGraham101) [inline Scripts/New-InitialDotNetCoreProjects.ps1]: I think we need to account for different project types, ie function apps

Comment (thelious1974) [reply Scripts/New-InitialDotNetCoreProjects.ps1]: I've added the ability to specify if you want a web or function project to the script.
We can add more types in later should we need to.

File: Scripts/New-InitialDotNetCoreProjects.ps1 [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    )
+
+    $solutionContent = Get-Content -Path $SolutionName
+
+    # Projects in the solution file have a distict structure:
+    # Project("<Project Type ID>") = "<Project Name>", "<Path To Project>", "<Project Guid>"
+    # The following searches for lines that match that pattern, and iterates over them,  placing
+    # the Project Guid into the project file.
+
+    $projects = $solutionContent | Select-String -Pattern "^Project\(.*\) = `".*`", `"(.*)`", `"(.*)`"$"
```
Comment (rivoric) [inline Scripts/New-InitialDotNetCoreProjects.ps1]: would using single quotes make this more readable

**Review by rivoric — APPROVED** [review]

File: Scripts/New-InitialDotNetCoreProjects.ps1 (hunk 2 above)
Comment (thelious1974) [reply Scripts/New-InitialDotNetCoreProjects.ps1]: Yes,  it would and does!

**Pull request merged**